### PR TITLE
Correct usages of custom error class

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
lib/ansible/modules/cloud/amazon/aws_direct_connect_link_aggregation_group.py

### CODE REVIEW

File: lib/ansible/modules/cloud/amazon/aws_direct_connect_link_aggregation_group.py [hunk 1]
```diff
@@ -193,7 +193,7 @@ def lag_exists(client, lag_id=None, lag_name=None, verify=True):
             failed_op = "Failed to describe DirectConnect link aggregation groups."
         raise DirectConnectError(msg=failed_op,
                                  last_traceback=traceback.format_exc(),
-                                 exception=e)
+                                 response=e)
```
Comment (s-hertel) [inline lib/ansible/modules/cloud/amazon/aws_direct_connect_link_aggregation_group.py]: Oh... I had intended to remove DirectConnectError and import it from ansible.module_utils.aws.direct_connect so the modules were sharing the code. Can you check if that fixes the problem? Nice catch.

Comment (Akasurde) [reply lib/ansible/modules/cloud/amazon/aws_direct_connect_link_aggregation_group.py]: OK.

**Comment by Akasurde** [comment]
@s-hertel Could you please take another look ? Thanks.

**Comment by Akasurde** [comment]
rebuild_merge

**Pull request merged**